tests/files/packet_centos8-calico-ha-ebpf.yml

```diff
@@ -0,0 +1,16 @@
+---
+# Instance settings
+cloud_image: centos-8
+mode: ha
+vm_memory: 3072
```

Comment:
Error message: `denied the request: spec.template.spec.domain.resources.requests.memory \\'3072\\': must be greater than or equal to 1M.`

Necessary to update here to
```
vm_memory: 3072Mi
```

Reply:
:facepalm: fixed now